components/external-services-mock/internal/subscription/handler.go

```diff
@@ -23,8 +23,7 @@ type handler struct {
 }
 
 type JobStatus struct {
-	ID    string `json:"id"`
```

Comment:
Why are we removing the `ID`?

Reply:
There is no ID in the response anymore:
Sample response:
{
	"status": "COMPLETED"
}